Classes/Emogrifier.php

```diff
@@ -402,6 +403,7 @@ protected function process(\DOMDocument $xmlDocument)
         }
 
         $this->copyCssWithMediaToStyleNode($xmlDocument, $xPath, $cssParts['media']);
+        restore_error_handler();
```

Comment:
Do the two lines need enclose this part of the code, or can we narrow it down so it encloses only the lines that actually do some XPath processing?